bin/ch-test

```diff
@@ -520,7 +520,7 @@ test_make_tag_from_path () {
     #   3) basicname:  the file name (Dockerfile)
     #   4) extension:  the file's extension (openmpi)
     #
-    # $basicname must be 'Build' or 'Dockerfile'; otherwise error.
+    # $basicname must be 'Build', 'Dockerfile'; otherwise error.
```

Comment:
Why was this changed to `,`? `or` seems to be the correct punctuation?

Reply:
That's a mistake. Thanks; I thought I changed it back.